src/ui/rideWindow.ts

```diff
@@ -115,6 +116,30 @@ export const rideWindow = window({
 				})
 			]
 		}),
+		groupbox({
+			text: "Ride name",
+			tooltip: "Edit the name of the ride",
+			content: [
+				label({
+					text: model._title,
+				}),
+				button({
+					text: "Rename ride",
+					height: buttonSize,
+					onClick: () => {
+						ui.showTextInput({
+							title: "Ride/attraction name",
+							initialValue: model._title.get(),
+							description: "Enter new name for this ride/attraction:",							
+							callback: (input) => {
+								context.executeAction("ridesetname", {ride: model._ride.get()?._id, name: input});
+								model._title.set(input);
+							}
+						})
```

Comment:
```suggestion
						});
```
Linting was giving a warning for this. (CI should fail for this though, weird that it doesn't. 🤔)

Reply:
corrected it :-)